test/pid_tests.cpp

```diff
@@ -43,23 +43,11 @@ TEST(ParameterTest, ITermBadIBoundsTest)
 {
   RecordProperty(
     "description",
-    "This test checks that the integral contribution is robust to bad i_bounds specification (i.e. "
+    "This test checks that the constructor throws an error for bad i_bounds specification (i.e. "
```

Comment:
Well technically it's "setGains" that throws. Can we trigger a throw with dyn reconf?

Reply:
never used dynamic reconfigure up to now, I'll see if I can write a test for that.

Reply:
@bmagyar I checked now how dynamic reconf works with the ROS-PID. So you mean if it is a good idea to throw an error in this case? I'm not sure how this is used in projects.. we could just give a RCLCPP_WARN message and forget the new parameters, but throw an error only if the constructor is called with i_min>i_max?

Reply:
@bmagyar I think we have to this in the controller. here we can only document the behavior